scripts/imagebuild.sh

```diff
@@ -25,11 +25,12 @@ dockerfile_dir="${base_dir}/images/federation-v2"
 
 echo "travis tag: ${TRAVIS_TAG}"
 echo "travis branch:${TRAVIS_BRANCH}"
-if [[ "${TRAVIS_TAG}" =~ ^[0-9]+\.[0-9]+\.[0-9]?\d+([-a-zA-Z0-9-]*)? ]]; then
-    echo "Pushing images with tag '${TRAVIS_TAG}'."
+if [[ "${TRAVIS_TAG}" =~ ^v([0-9]\.)+([0-9])[-a-zA-Z0-9]*([.0-9])* ]]; then
```

Comment:
Something changed about this regex - it's now looking for `1.1-blah` not `1.1.1-blah`.